### DISCUSSION TITLE
2.11.5.3

### DISCUSSION THREAD

**Discussion by rbayet** [Smile-SA/elasticsuite#3224] [General]
## Emergency maintenance release

Releases [2.10.19.2](https://github.com/Smile-SA/elasticsuite/releases/tag/2.10.19.2) and [2.11.5.2](https://github.com/Smile-SA/elasticsuite/releases/tag/2.11.5.2) published yesterday can cause an issue if you minify your JS sources, preventing JS scripts from working in Magento's admin area.
Thanks @thomas-kl1 for a quick reaction !

## :bug: Fixes

* [Quality] Remove invalid characters in new JS files by @thomas-kl1 in https://github.com/Smile-SA/elasticsuite/pull/3222

**Full Changelog**: https://github.com/Smile-SA/elasticsuite/compare/2.11.5.2...2.11.5.3

<hr /><em>This discussion was created from the release <a href='https://github.com/Smile-SA/elasticsuite/releases/tag/2.11.5.3'>2.11.5.3</a>.</em>

**Comment by ThisIsRuddy** [comment]
Spent hours trying isolate what module was causing the issue, after diffing locks and incrementally re-adding the l changes

So ive literally just this minute found it was the elasticsuite module, I arrive here and you patched it 20 mins ago 😅
